acceptance-tests/src/main/java/io/blueocean/ath/WebElementUtils.java

```diff
@@ -27,7 +27,7 @@ public static void clearText(WebElement element) {
      */
     public static void setText(WebElement element, String text) {
         checkTextElement(element);
-        element.sendKeys(Keys.chord(Keys.CONTROL, "a"));
+        element.clear();//sendKeys(Keys.chord(Keys.CONTROL, "a"));
```

Comment:
is there a reason it was doing `CTRL-a` instead of `clear` before? this definitely doesn't work on Mac ;)

Reply:
ctrl-a goes back to the start of a text cell (emacs keys - may they never die) which I gather is not the intent here?

Reply:
Sorry, somehow I completely missed the presence of `clear()` in the docs. That is much easier. :)